src/context/Context.ts

```diff
@@ -1,14 +1,15 @@
+import { IdService, LoggerService, SecurityService } from '../Services';
 import { EventSessionFactory } from './events/Session';
-import { Gallery } from './gallery';
-import { MongoService } from './mongo';
+import { GalleryService } from '../Services';
+import { MongoService } from '../Services';
 
 export type ContextType = {
-  $id: any;
-  $logger?: any;
-  $image?: any;
-  $file?: any;
-  $security?: any;
+  $id: IdService;
+  $logger: LoggerService;
+  $security: SecurityService;
   $mongo: MongoService;
+  // $image?: any;
+  // $file?: any;
```

Comment:
Remove log

Reply:
Interesting, need to check. Im sure I brought them back in and typed them.